src/lib/load-layersets.js

```diff
@@ -86,14 +88,16 @@ export async function loadLayerSetById (id, options) {
   return layerSet
 }
 
-export function loadLayerSets () {
+export async function loadLayerSets () {
   // we don't need to login anymore, but the function is still used to
   // get the list of public maps
   const body = JSON.stringify({
-    username: 'marko@voorhoede.nl',
-    password: 'DikkeDoei123',
+    username: 'anonymous@rws.nl',
```

Comment:
un and pw specified in code?

Reply:
Not anymore. I removed them. The service is anonymous.